snare

```diff
@@ -0,0 +1,72 @@
+#! /usr/bin/env python3.7
+
+# Copyright (c) 2019 King's College London, created by the Software Development
+# Team <http://soft-dev.org/>
+#
+# Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
```

Comment:
Is it me, or does it look odd to split `<...>` over multiple lines? It took me a while to parse.

(Not super-important)

Reply:
When it's that long, I don't think there's much choice.